de.metas.swat/de.metas.swat.base/src/main/java/de/metas/inoutcandidate/api/impl/ReceiptScheduleDAO.java

```diff
@@ -230,11 +230,7 @@ else if (tableID == InterfaceWrapperHelper.getTableId(I_M_InOutLine.class))
 				schedules = ImmutableSet.copyOf(retrieveRsForInOutLine(inoutLine));
 			}
 		}
-		else
```

Comment:
why this was deleted?

Reply:
Because if is an else, we should have other listener.
Anyway that was empty brakets

Reply:
i would prefer to make the `schedules` final.